wiki/en/Installation-for-Linux.md

```diff
@@ -30,7 +30,7 @@ For installers on other distributions, see their package managers and [Repology]
 
 ### Configure JACK with QjackCtl
 
-Jamulus clients need [JACK](https://jackaudio.org/) to run, but you need to configure that first. The recommended method is to use `QjackCtl`.
+Jamulus Clients need [JACK](https://jackaudio.org/) to run, but you need to configure that first. The recommended method is to use `QjackCtl`.
```

Comment:
Again, unrelated to this change...  Is `qJackCtl` _still_ the recommended tool?

Reply:
Do you imply you know of others?

Reply:
> Do you imply you know of others?

Cadence, for exemple.

Reply:
Ok, but I mean is that better than qJackCtl? Should we recommend it? Also, I seem to recall the .deb will recommend (or even install?) qJackCtl when you apt install - or am I imagining that?

Reply:
I will not say better than  QjackCtl but there is no MacOs version for Cadence for now.

Reply:
I guess it's one to watch though. qJackCtl is a bit complicated, so if there was a "bare bones" configuration UI, that might be better.